backend-api/run-tests-locally.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -eu
+
+stack_name=${1:-mob-async-backend}
+
+echo Running test against "${stack_name}"
```

Comment:
Nit: 
```suggestion
echo "Running test against ${stack_name}"
```

Reply:
is that a self-nit?

Reply:
Always working on becoming a better dev.